ftw/envoy-config.yaml

```diff
@@ -37,10 +37,10 @@ static_resources:
                           value: |
                             {
                               "rules": [
-                                "Include coraza.conf-recommended.conf",
-                                "Include ftw-config.conf",
+                                "Include @recommended-conf",
+                                "Include @ftw-conf",
                                 "Include crs-setup.conf.example",
```

Comment:
What is the reasoning behind not converting also `crs-setup.conf.example` into a keyword? I feel that it may even have a broader usage than the one intended for demo purposes (`@crs-setup-demo-conf`)

Reply:
I guess the reason is just me being silly. Would you please add it in a next PR? 🙌

Reply:
Sure, no worries! Thanks 🙏